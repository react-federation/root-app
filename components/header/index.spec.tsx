import { BrowserRouter } from 'react-router-dom'
import setup from 'utils/test/setup'
import Header from '.'

describe('Header', () => {
  it('should match snapshot', () => {
    const { asFragment } = setup(Header, {}, BrowserRouter)

    expect(asFragment()).toMatchSnapshot()
  })
})
