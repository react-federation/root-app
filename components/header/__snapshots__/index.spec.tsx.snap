// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  background-color: purple.100;
  padding-top: 4px;
  padding-bottom: 4px;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
}

<header
    class="emotion-0"
  >
    <div
      class="container emotion-1"
    >
      <div
        class="emotion-2"
      >
        <a
          href="/"
        >
          <h2
            class="chakra-heading emotion-2"
          >
            Cryptos âœ¨ðŸ¤‘
          </h2>
        </a>
      </div>
      <a
        href="/cryptos"
      >
        Cryptos price
      </a>
    </div>
  </header>
</DocumentFragment>
`;
